apps/wallet/src/ui/app/index.tsx

```diff
@@ -159,65 +160,75 @@ const App = () => {
+				<StorageMigrationPage />
+			) : (
+				<Routes>
+					<Route path="restricted" element={<RestrictedPage />} />
+					<Route path="/*" element={<HomePage />}>
+						<Route path="apps/*" element={<AppsPage />} />
+						<Route path="kiosk" element={<KioskDetailsPage />} />
+						<Route path="nft-details" element={<NFTDetailsPage />} />
+						<Route path="nft-transfer/:nftId" element={<NftTransferPage />} />
+						<Route path="nfts/*" element={<AssetsPage />} />
+						<Route path="onramp" element={<OnrampPage />} />
+						<Route path="receipt" element={<ReceiptPage />} />
+						<Route path="send" element={<TransferCoinPage />} />
+						<Route path="send/select" element={<CoinsSelectorPage />} />
+						<Route path="stake/*" element={<Staking />} />
+						<Route path="tokens/*" element={<TokenDetailsPage />} />
+						<Route path="transactions/:status?" element={<TransactionBlocksPage />} />
+						<Route path="*" element={<Navigate to="/tokens" replace={true} />} />
+					</Route>
+					<Route path="accounts/*" element={<AccountsPage />}>
+						<Route path="welcome" element={<WelcomePage />} />
+						<Route path="add-account" element={<AddAccountPage />} />
+						<Route path="import-ledger-accounts" element={<ImportLedgerAccountsPage />} />
+						<Route path="import-passphrase" element={<ImportPassphrasePage />} />
+						<Route path="import-private-key" element={<ImportPrivateKeyPage />} />
+						<Route path="manage" element={<ManageAccountsPage />} />
+						<Route path="protect-account" element={<ProtectAccountPage />} />
+						<Route path="backup/:accountSourceID" element={<BackupMnemonicPage />} />
+						<Route
+							path="qredo-connect/*"
+							element={
+								<PageMainLayout>
+									<Outlet />
+								</PageMainLayout>
+							}
+						>
+							<Route path=":requestID" element={<QredoConnectInfoPage />} />
+							<Route path=":id/select" element={<SelectQredoAccountsPage />} />
+						</Route>
+						<Route path="export/:accountID" element={<ExportAccountPage />} />
+						<Route path="export/passphrase/:accountSourceID" element={<ExportPassphrasePage />} />
+						<Route path="forgot-password" element={<ForgotPasswordPage />}>
+							<Route index element={<ForgotPasswordIndexPage />} />
+							<Route path="recover" element={<RecoverPage />} />
+							<Route path="recover-many" element={<RecoverManyPage />} />
+							<Route path="warning" element={<ResetWarningPage />} />
+							<Route path="reset" element={<ResetPasswordPage />} />
+						</Route>
+					</Route>
+					<Route path="/dapp/*" element={<HomePage disableNavigation />}>
+						<Route path="connect/:requestID" element={<SiteConnectPage />} />
+						<Route path="approve/:requestID" element={<ApprovalRequestPage />} />
+					</Route>
+					{process.env.NODE_ENV === 'development' ? (
+						<Route path="/accounts-dev" element={<AccountsDev />} />
+					) : null}
+				</Routes>
+			)}
+			<div id="overlay-portal-container"></div>
```

Comment:
id probably move these to `src/ui/index.tsx`

Reply:
i just meant these two divs

Reply:
I ended up moving the whole think there seemed clearer